tac/project-updates/2025/2025-annual-Hiero.md

```diff
@@ -0,0 +1,82 @@
+
+## Goals
+
+### Performance Against Prior Goals
+
+Our initial target was to move all Hedera repositories to Hiero.
+While we've mostly finished this transition, we've initiated the process of refactoring the code to a more vendor-agnostic and unbranded architecture.
+Our main focus is our SDKs in addition to discussing workflows, roadmaps, and patterns.
+Information and current state on the transition from Hedera can be found [here](https://github.com/hiero-ledger/hiero/blob/main/transition.md).
+Information about our work on making the SDKs vendor neutral can be found [here](https://github.com/hiero-ledger/hiero/blob/main/blog/2025-03-17-sdk-workflows.md).
+
+With all of this work in progress, we plan to release the initial minor versions of all projects in the near future.
+Next, we plan a major release that will contain all the work from moving the code to a more vendor neutral release.
+
+Another main goal is the onboarding of community projects from the Hiero ecosystem.
+At the moment, we moved 3 community projects to Hiero and a new proposal is in the pipeline.
+Information about the onboarding of new projects can be found [here](https://github.com/hiero-ledger/hiero/blob/main/community-transition.md).
+
+On the project's management front, the team is focusing on evaluating and analyzing key project health metrics that track our progress in fostering a thriving and diverse community.
+These metrics enable us to assess the project's performance by measuring contributions, code velocity, and issue response and resolution times.
+
+Our metrics report can be found [here](https://docs.google.com/document/d/e/2PACX-1vRxg-3wU-eg0EWEhAu9SRab_sXwJBy5YDQqoeV6BLq0FnVnoO4V41kPYEDYJovtns6tpq9gkaSaSzlO/pub).
+
+### Next Year's Goals
+
+Hiero is a rapidly growing project that is gaining significant traction. We anticipate continued growth in our community's diversity, enabling the adoption of more community-driven components.
+
+The project is expected to see a considerable growth in the number of maintainers, committers, and members of the TSC.
+
+Our vendor-neutral releases will provide the project with enhanced flexibility and adaptability, supporting its long-term success.
+
+As a project, we expect Hiero to continue advancing in performance, scalability, and, most importantly, trust.
+We remain committed to continue growing our community engagement by encouraging contributions to support both our current and upcoming SDKs, as well as helping to build a more comprehensive TCK with richer test coverage to ensure robust end-to-end validation.
+Additionally, we are focused on achieving faster finality, lower latency, and continuously increasing capacity to support larger transactions.
+
+### Help Required
+
+As outlined in the following section, we would like to initiate a discussion with the TAC regarding the transition of the project to the graduated state.
+We would greatly appreciate any feedback or guidance on this process.
```

Comment:
Besides the incubation exit criteria, also take a look at the project lifecycle governance document that adds information on additional expectations that might be needed. Typically, the process would be to create a PR under the project-proposals repo that outlines the request to move to graduated and how the project meets the criteria for graduation.

Reply:
Thank you Tracy, I think that is this document?
https://lf-decentralized-trust.github.io/governance/governing-documents/project-lifecycle.html
I will double check